Add unicode symbols to sound devices page (#552)
jamulussoftware/jamulussoftware.github.io: _posts/2021-01-05-Jamulus-Sound-Devices.md

## Audio devices known to work with Jamulus

Your device must support a sample rate of *48 kHz (DVD quality)* - which many devices do - and should allow you to listen to the computer’s audio signal. On Windows it’s recommended to choose a device with a *native ASIO driver* - not ASIO4ALL.
Whether the following devices work or not can *depend on your operating system* (Linux, MacOS, Windows, etc.). We've added symbols after each OS to make judging easier:

✅: This device works reasonably well with Jamulus. You might need to set up some things, but sound quality and latency are fine.

> 😐: This device works so-so and there might be some difficulties. If you want to buy a new device, don't buy this one. There might be similar devices available which work better.

Reviewer: ```suggestion
😐: This device works, but there might be some difficulties. If you want to buy a new device, don't buy this one. There might be similar devices available which work better.
```
